test-tools/wamr-ide/README.md

```diff
@@ -33,14 +33,14 @@ The WAMR-IDE is an Integrated Development Environment to develop WebAssembly app
 
 #### 3. Build docker images
 
-We have 2 docker images which should be built or loaded on your host, `wasm-toolchain` and `wasm-debug-server`. To build these 2 images, please enter the `WASM_Source_Debug_Server/Docker` & `WASM_Toolchain/Docker`, then execute the `build_docker_image` script respectively.
+We have 2 docker images which should be built or loaded on your host, `wasm-toolchain` and `wasm-debug-server`. To build these 2 images, please enter the `WASM-Debug-Server/Docker` & `WASM-Toolchain/Docker`, then execute the `build_docker_image` script respectively.
 
 Windows (powershell):
 
 ```batch
 $ cd .\WASM-Toolchain\Docker
 $ ./build_docker_image.bat
-$ cd .\WASM-Source-Debug-Server\Docker
+$ cd .\WASM-Debug-Server\Docker
```

Comment:
L42 and L44, we had better change `./` to `.\` as it is on Windows platform. Could you please help change them?

Reply:
done